app/src/main/java/org/mozilla/fenix/ext/Activity.kt

```diff
@@ -48,3 +56,59 @@ fun Activity.breadcrumb(
         )
     )
 }
+
+/**
+ *  Opens Android's Manage Default Apps Settings if possible.
+ */
+fun Activity.openSetDefaultBrowserOption() {
+    when {
+        Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q -> {
+            getSystemService(RoleManager::class.java).also {
+                if (it.isRoleAvailable(RoleManager.ROLE_BROWSER) && !it.isRoleHeld(
+                        RoleManager.ROLE_BROWSER
+                    )
+                ) {
+                    startActivityForResult(
+                        it.createRequestRoleIntent(RoleManager.ROLE_BROWSER),
+                        REQUEST_CODE_BROWSER_ROLE
+                    )
+                } else {
+                    navigateToDefaultBrowserAppsSettings()
+                }
+            }
+        }
+        Build.VERSION.SDK_INT >= Build.VERSION_CODES.N -> {
+            navigateToDefaultBrowserAppsSettings()
+        }
+        else -> {
+            (this as HomeActivity).openToBrowserAndLoad(
+                searchTermOrURL = SupportUtils.getSumoURLForTopic(
+                    this,
+                    SupportUtils.SumoTopic.SET_AS_DEFAULT_BROWSER
+                ),
+                newTab = true,
+                from = BrowserDirection.FromSettings
+            )
+        }
+    }
+}
+
+private fun Activity.navigateToDefaultBrowserAppsSettings() {
```

Comment:
Where's my kDoc?

Reply:
As it's private helper function I think it wasn't need it, but we can add it too.